scripts/perfmon

```diff
@@ -500,7 +503,10 @@ class Variable(VariableConfig, VariableState):
             return # nothing to do
         self.active = active
         if active:
-            VariableState.__init__(self) # reset when reactivating
+            # reset when reactivating
+            self.value = None
+            self.timeof_last_alarm = time.time() - self.alarm_auto_inhibit_period
+            self.trigger_down_counter = self.alarm_trigger_period
```

Comment:
this duplicates code that previously wasn't duplicated.

Reply:
Also for passing the pytype check.
The original code is a class, and pytype complained that the class has no `alarm_auto_inhibit_period` and `alarm_trigger_period` attributes.